### ISSUE TITLE
Uploading Profile Pictures and Banner Pictures on Edit Profile

### ISSUE THREAD

**Issue by MarkoBraticevic** [PrimalHQ/primal-ios-app#87]
### Description
Uploading pictures to update a profile picture or a banner picture doesn't automatically change on the app. The previously used profile and banner pictures will remain. The profile picture on each post made by the user will be changed to the updated picture, but the profile page won't update. 

### Steps to reproduce
- Go to the profile page and select "Edit Profile"
- Upload a new picture for your profile and for your banner pictures, and then save that information. 
- See if the profile page is updated with the new uploaded images, along with the profile picture on each post.

### Expected Behavior
The profile page should be updated as soon as I save the edited profile's information. We should see the new uploaded profile and banner pictures right away.

### Actual Behavior
 - The profile page will keep the previous profile and banner pictures, and not update to the new saved profile pictures.

### Additional context
If you edit your profile, save the information and then search the account on the web app, it will show you the new updated profile information, so the profile is being edited, it's just not updating on the iOS app.

**Comment by nearlymad** [comment]
I agree this doesn't work correctly. I have to close the app and reopen for it to update.
